task_02/index.html

```diff
@@ -1,11 +1,195 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
This is not react, you are not closing meta tags this way.

https://www.w3schools.com/tags/tag_meta.asp

Reply:
Will be fixed.